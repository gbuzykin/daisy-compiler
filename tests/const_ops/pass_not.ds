#if ~2678i32 != -2679i32
#error invalid result
#endif
#if ~2678u32 != 4294964617u32
#error invalid result
#endif
#if ~-2678i32 != 2677i32
#error invalid result
#endif
#if ~4294964618u32 != 2677u32
#error invalid result
#endif
