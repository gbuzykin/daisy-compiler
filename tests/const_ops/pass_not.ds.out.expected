./const_ops/pass_not.ds: info: warnings 0, errors 0
